xc-admin/packages/xc-admin-common/src/governance_payload/ExecutePostedVaa.ts

```diff
@@ -103,5 +109,36 @@ export function decodeExecutePostedVaa(
     }
   );
 
-  return { header, instructions };
+  return { targetChainId: header.targetChainId, instructions };
+}
+
+/** Encode ExecutePostedVaaArgs */
+export function encodeExecutePostedVaa(src: ExecutePostedVaaArgs): Buffer {
+  // PACKET_DATA_SIZE is the maximum transactin size of Solana, so our serialized payload will never be bigger than that
+  const buffer = Buffer.alloc(PACKET_DATA_SIZE);
```

Comment:
this code comment seems wrong because the code doesn't enforce this size constraint. I could theoretically make `src` have a bunch of `instructions` in it such that the serialized version of the message is bigger than PACKET_DATA_SIZE.

Reply:
This part is a little awkward, I don't know the length before I encode it.
I'll try to improve the comment.